lib/emotions/emotional.rb

```diff
@@ -136,5 +137,6 @@ def #{emotion}_about(emotive)
         RUBY
       end
     end
+    # rubocop:enable MethodLength
```

Comment:
I think this line is supposed to go between lines `138` and `139` (right after the `define_emotion_methods`’s `end` keyword).